docs/Getting-Started/Create-bootable-media/bootable-usb-using-linux.md

```diff
@@ -0,0 +1,111 @@
+# Create a Bootable Pop!_OS USB in Linux
+
+Use this guide to create Pop!_OS bootable USB media from Linux:
+
+- Install ISO flashing software.
+- Download the Pop!_OS.iso file.
+- Verify the ISO.
+- Flash the ISO to your USB storage device.
+
+---
+## I. Install Popsicle
+
+![Popsicle Icon](/images/create-bootable-usb-linux/popsicle-icon.png)
+
+Create a bootable USB drive for Pop!_OS using Popsicle. Popsicle is a Linux utility for flashing multiple USB devices in parallel, written in Rust. You can install Popsicle via flatpak or AppImage.
+### AppImage
+
+1. Download the Popsicle AppImage [from this repo.](https://github.com/pop-os/popsicle/releases/latest)
```

Comment:
I would move the word "from" out of the link. Also wonder if this could be worded more user-friendily than "repo."

Reply:
Maybe: from "[the Popsicle Github page](link)."

Even better if we can link directly to the releases section or the AppImage download vs. just the main page.

Reply:
"Github page" doesn't sound good to me. Let's do `from the [Popsicle repository](link)`. Also future-proofs in case we ever move to gitlab.